react/deprecated/Modal/ModalEffects.ts

```diff
@@ -8,38 +8,46 @@ import { isFlagshipApp } from 'cozy-device-helper'
 
 import { useSetFlagshipUI } from '../../hooks/useSetFlagshipUi/useSetFlagshipUI'
 import { getFlagshipMetadata } from '../../hooks/useSetFlagshipUi/helpers'
+import useColorScheme from '../../hooks/useColorScheme'
 
 const getTopBackground = (theme: Theme, cozyBar: Element | null): string =>
   (cozyBar && getComputedStyle(cozyBar).getPropertyValue('background-color')) ||
   theme.palette.background.paper
 
-const getTopTheme = (cozyBar: Element | null): 'light' | 'dark' =>
+const getTopTheme = (
+  cozyBar: Element | null,
+  isDark: boolean
+): 'light' | 'dark' =>
   getFlagshipMetadata().immersive ||
-  (cozyBar && cozyBar.classList.contains('coz-theme-primary'))
+  (cozyBar && cozyBar.classList.contains('coz-theme-primary')) ||
```

Comment:
est-ce que c'est toujours vrai après les nombreuses modif sur la cozy-bar ? @cballevre

Reply:
Probably no longer true with v12, but I haven't tested it. On the other hand, you should still need it for previous versions, such as v7, which is still available on certain apps.

Reply:
un petit commentaire dans le code serait bienvenu histoire qu'on ne se repose pas la question ?

Reply:
Complètement 👍 j'ai rajouté